src/qt/window.cpp

```diff
@@ -350,6 +350,8 @@ bool wxWindowQt::Create( wxWindowQt * parent, wxWindowID id, const wxPoint & pos
     // that a generic control, like wxPanel, is being created, so we need a very
     // simple control as a base:
 
+    wxSize size_ = size;
```

Comment:
Minor, but `size_` is not very readable. Let's rename it to `initialSize` maybe?